libs/fabric/src/lib/components/text-field/base-text-field.component.ts

```diff
@@ -68,6 +68,7 @@ export class FabBaseTextFieldComponent extends ReactWrapperComponent<ITextFieldP
   @Output() readonly onChange = new EventEmitter<{ event: Event; newValue?: string }>();
   @Output() readonly onBeforeChange = new EventEmitter<{ newValue: any }>();
   @Output() readonly onNotifyValidationResult = new EventEmitter<{ errorMessage: string; value: string | undefined }>();
+  @Output() readonly onBlur = new EventEmitter<{ event: Event }>();
```

Comment:
Please move these changes to a separate PR. It makes it easier to maintain and create the changelog (this part of the PR looks fine as it is, so if you move it it can get it as-is)

Reply:
OK, I'll try to do it but.... I don't know so well GIT :)

Reply:
Basically create a new branch from `master` and make the same changes there (or you can cherry-pick the commit using `git cherry-pick 95e1991` when you're on the new branch.
Then create a new PR from it.